scripts/learning/dissecLearning.js

```diff
@@ -19,27 +19,34 @@ const { createLogger } = require('../../src/targets/services/helpers/utils')
  * @param {Object[]} validationSet - The dataset used for measuring performances
  * @returns The accuracy of the model on the validation set
  */
-const dissecLearning = async (
+const dissecLearning = async ({
   client,
   cutoffDate,
   validationSet,
-  uri = 'http://test1.localhost:8080'
-) => {
+  uri = 'http://test1.localhost:8080',
+  pretrained = false
+}) => {
   const { log } = createLogger()
 
   // Create the tree, exclude the querier from contributors and aggregators
   const aggregationNodes = JSON.parse(
     fs.readFileSync(`${process.cwd()}/assets/webhooks.json`).toString()
   )
-  const querierWebhooks = aggregationNodes.filter(e => e.label === uri)[0]
-  const aggregatorsWebhooks = aggregationNodes.filter(e => e.label !== uri)
-  const contributorsWebhooks = aggregatorsWebhooks
-
-  const contributors = createTree(
-    querierWebhooks,
-    aggregatorsWebhooks,
-    contributorsWebhooks
-  )
+
+  const treeStructure = [
```

Comment:
Could you detail a bit in comment the tree structure and the expected params?

Reply:
Addressed in @b4e99c84ebab9eecf66dcb10e65723691f572299